Extension/src/LanguageServer/client.ts

```diff
@@ -58,6 +58,7 @@ const localize: nls.LocalizeFunc = nls.loadMessageBundle();
 let ui: UI;
 let timeStamp: number = 0;
 const configProviderTimeout: number = 2000;
+let clientCount: number = 0;
```

Comment:
Is this the same as this.clients.Count?

Reply:
I am not sure, does that attribute keep a running count of clients as they are initialized?

Reply:
Maybe rename it `initializedClientCount` to make that clear?

Reply:
Yeah, it should just be renamed so others aren't confused, like I was.

Reply:
in other words, the ClientColleciton has a count of the clients already, but not whether they're finished initializing yet.